app/pages/site.py

```diff
@@ -319,7 +324,9 @@ def update_interferogram(click_data, target_id, zoom, bounds):
             )
         return url, info_text
     else:
-        print('Layer does not exist')
+        logger.info('Failed to load interferogram: %s_HH_%s_HH.adf.wrp.geo.tif',
```

Comment:
Flake8 warns on this line (line too long)

Reply:
Resolved in previous commit